website/.eslintrc.json

```diff
@@ -0,0 +1,71 @@
+{
```

Comment:
we shouldn't redeclare a configuration.
The goal of the monorepository is to share the linter, formatter configuration across all the modules, not to fork/duplicate

Reply:
there was an issue with it:
```
Parsing error: Cannot read file '/home/anders/podman-desktop/website/website/tsconfig.json'
```

Reply:
Not sure why it needed an extra period ?

```json
    "project": [
      "packages/*/tsconfig.json",
      "./website/tsconfig.json",
      "./extensions/*/tsconfig.json",
      "./tests/tsconfig.json",
      "./tools/tsconfig.json"
    ]

```

Reply:
Did a workaround instead, apparently the feature is missing from json (only available in js)

`"tsconfigRootDir": __dirname`